masking_and_moments/masking_and_moments_guide.ipynb

```diff
@@ -0,0 +1,918 @@
+    "\n",
+    "ax = plt.subplot(projection=unmasked_moment1.wcs)\n",
+    "ax = plt.subplot()\n",
+    "im = ax.imshow(unmasked_moment1.value, origin='lower', cmap='coolwarm')\n",
+    "cbar = plt.colorbar(im)\n",
+    "cbar.set_label('Centroid (km/s)')\n",
+    "\n",
+    "ax.set_ylabel('Declination')\n",
+    "ax.set_xlabel('Right Ascension')"
+   ]
+  },
+  {
+   "source": [
+    "Spectra dominated by noise lead to extremely large outliers!"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "### Line width -- (Square root) Moment 2 ###\n",
+    "\n",
+    "The second moment measure the [_variance_]((https://en.wikipedia.org/wiki/Variance)) of the spectrum around the centroid $v_0$, weighted by the intensity:\n",
+    "$$\n",
+    "M_2 = \\frac{\\int_{\\mathrm{line}} (v-v_0)^2 T(v)\\,dv}{\\int_{\\mathrm{line}} T(v)\\,dv} \\approx \\frac{\\sum_{i\\in\\{\\mathrm{line}\\}} (v_i-M_1)^2 T(v_i)\\,\\delta v}{M_0}.\n",
+    "$$ \n",
+    "\n",
+    "\n",
+    "For a Gaussian spectrum, $M_2\\approx\\sigma_v^2$, and would be exact without noise.\n",
+    "\n",
+    "While the moment 2 is often referenced as the line width, the line width is formally the square root of the second moment:\n",
+    "$$\n",
+    "\\sigma_c \\approx \\sqrt{M_2}.\n",
+    "$$\n",
+    "\n",
+    "In spectral-cube, these two quantities are labeled separately to avoid confusion. The second moment can be calculated with:"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "masked_moment2 = masked_cube.moment2()"
+   ]
+  },
+  {
+   "source": [
+    "As the warning message explains, the _line width_ can be calculated with:"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "masked_linewidth = masked_cube.linewidth_sigma()\n",
+    "\n",
+    "ax = plt.subplot(projection=masked_linewidth.wcs)\n",
+    "ax = plt.subplot()\n",
+    "im = ax.imshow(masked_linewidth.value, origin='lower', cmap='coolwarm')\n",
+    "cbar = plt.colorbar(im)\n",
+    "cbar.set_label('Line Width (km/s)')\n",
+    "\n",
+    "ax.set_ylabel('Declination')\n",
+    "ax.set_xlabel('Right Ascension')"
+   ]
+  },
+  {
+   "source": [
+    "We see that the brightest regions in the integrated intensity map corresponds to the largest line widths. However, we note that the edges of the mask likely contain fewer spectral channels, which can artificially produce narrower line widths. Caution should be used when interpreting the line width where $\\sim4$ or fewer channels are included in the mask.\n",
+    "\n",
+    "Another common convention is to define the full-width-half-max line width:\n",
+    "$$\n",
+    "\\textrm{FWHM}=\\sqrt{8 {\\rm ln} 8} \\sigma_v \\ \\approx 2.35 \\sigma_v\n",
```

Comment:
```suggestion
    "\\textrm{FWHM}=\\sqrt{8 {\\rm ln} 2} \\sigma_v \\ \\approx 2.35 \\sigma_v\n",
```

Reply:
Had it fixed locally but hadn't pushed it at the time.